CHANGELOG.md

```diff
@@ -27,11 +27,14 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Removed
 
-- Nothing.
+- [#33](https://github.com/zendframework/zend-di/pull/32) removes the implementation of
+  `Psr\Log\LoggerAwareInterface` from `Zend\Di\CodeGenerator\InjectorGenerator` that
+  was introduced with [#32](https://github.com/zendframework/zend-di/pull/32)
```

Comment:
I think we should remove it, because #32 is not going to be released, because this PR replaces it. The same in `Fixed` section.
Notes in `Added` section are enough. But this is my option and maybe I am wrong?

Reply:
I agree, in fact I was a bit unsure about it. I'd like to remove these entries for the same reason.
@Ocramius any objections?

Reply:
I agree with @webimpress. Instead, note that #32 and #34 provide the ability to log generator processing, and give a short example of how that can be enabled.